test/integration/dbt_test.py

```diff
@@ -81,17 +76,16 @@ def test_update_tables(
     tests don't run until after the ETL has completed and the Parquet files are
     available.
     """
+    args = [
+        "dbt_helper",
+        "update-tables",
+        # "--schema",  # Uncomment when we have schema-preserving updates
+        "all",
+    ]
+    if dbt_target == "etl-full":
+        args.append("--row-counts")
```

Comment:
Only run with the `--row-counts` flag if we're using the `full-etl` outputs -- otherwise we'll get a bunch of spurious row count updates in CI.